airflow/logging_config.py

```diff
@@ -54,8 +58,8 @@ def configure_logging():
     else:
         logging_class_path = 'airflow.config_templates.' \
                              'airflow_local_settings.DEFAULT_LOGGING_CONFIG'
-        logging_config = import_string(logging_class_path)
-        log.debug('Unable to load custom logging, using default config instead')
+        logging_config = DEFAULT_LOGGING_CONFIG
```

Comment:
Where is this coming from? Don't we need: `import_string(logging_class_path)` line ?